api/src/main/resources/leonardo.yaml

```diff
@@ -151,13 +144,6 @@ paths:
           required: false
           schema:
             type: string
-        - in: query
```

Comment:
No longer found in Leo's swagger:
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml